docker/image/whyis-server-deps/Dockerfile

```diff
@@ -5,7 +5,12 @@ RUN puppet module install\
 	puppet-python --version 3.0.1
 
 COPY /docker/image/whyis-server-deps/install.pp /
+COPY /puppet/files/etc/init.d/celeryd /tmp/celeryd
```

Comment:
Just copy it to where it belongs here instead of adding another stanza to the puppet.